README.md

```diff
@@ -57,22 +57,122 @@ Example tag/value parsing usage:
 The `examples` directory contains several code samples:
 
 * `parse_tv.py` is an example tag/vlue parsing uage.
-  Try running `python parse_tv.py 'data/SPDXSimpleTag.tag' `
+  Try running `python parse_tv.py '../data/SPDXSimpleTag.tag' `
```

Comment:
Yes. For running this sample code either you have to be in `examples` directory and then try `python parse_tv.py '../data/SPDXSimpleTag.tag'` or in the `main` directory and then try `python examples/parse_tv.py 'data/SPDXSimpleTag.tag'` .